app/src/main/java/in/testpress/testpress/ui/WebViewActivity.java

```diff
@@ -0,0 +1,282 @@
+package in.testpress.testpress.ui;
```

Comment:
Is this activity generic? Can this be reused for force update user detail webview?

Reply:
Yes, It's generic. It can handle any URL. That's why I named it as webviewActivity.